test/e2e/suites/e2e/self_hosted_test.go

```diff
@@ -0,0 +1,43 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
```

Comment:
As in https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/4526db5675ddaf7e62ab8017af7cb1ff46d1b201/test/e2e/shared/cluster.go#L29
```suggestion
	capie2e "sigs.k8s.io/cluster-api/test/e2e"
```